### PR TITLE
Add ordering by `createdDateTime` for Named Versions and by `acquiredDateTime` for Briefcases

### PR DESCRIPTION
**Pull request by arminasbrazenas**
- Add ordering by `createdDateTime` for Named Versions
- Add ordering by `acquiredDateTime` for Briefcases

### CODE REVIEW

File: tests/imodels-clients-tests/src/integration/management/BriefcaseOperations.test.ts [hunk 1]
```diff
@@ -27,6 +29,35 @@ describe("[Management] BriefcaseOperations", () => {
...
+      authorization,
+      iModelId: testIModel.id,
+    }));
+    expect(testIModelBriefcases.length).to.equal(1);
+
+    const testIModelsClient = container.get(TestIModelsClient);
+    for (let i = 1; i <= 2; i++) {
+      const briefcase = await testIModelsClient.briefcases.acquire({
+        authorization,
+        iModelId: testIModel.id,
```
[the remark this answers is not in the document]
Comment (arminasbrazenas) [reply tests/imodels-clients-tests/src/integration/management/BriefcaseOperations.test.ts:43]: Maybe we could acquire multiple briefcase in the reusable iModel?

File: tests/imodels-clients-tests/src/integration/management/BriefcaseOperations.test.ts [hunk 2]
```diff
@@ -105,7 +139,10 @@ describe("[Management] BriefcaseOperations", () => {
       authorization,
       iModelId: testIModel.id,
       urlParams: {
-        $top: 1
+        $top: 1,
+        $orderBy: {
+          property: BriefcaseOrderByProperty.AcquiredDateTime
+        }
```
[the remark this answers is not in the document]
Comment (arminasbrazenas) [reply tests/imodels-clients-tests/src/integration/management/BriefcaseOperations.test.ts]: Not needed, removed. Thanks

File: tests/imodels-clients-tests/src/integration/management/BriefcaseOperations.test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (arminasbrazenas) [reply tests/imodels-clients-tests/src/integration/management/BriefcaseOperations.test.ts]: Acquired several briefcases in reusable iModel.

File: tests/imodels-clients-tests/src/integration/management/BriefcaseOperations.test.ts [hunk 3]
```diff
@@ -27,6 +29,35 @@ describe("[Management] BriefcaseOperations", () => {
 
     const reusableTestIModelProvider = container.get(ReusableTestIModelProvider);
     testIModel = await reusableTestIModelProvider.getOrCreate();
+
+    testIModelBriefcases = await toArray(iModelsClient.briefcases.getRepresentationList({
```
[the remark this answers is not in the document]
Comment (arminasbrazenas) [reply tests/imodels-clients-tests/src/integration/management/BriefcaseOperations.test.ts]: Acquired several briefcases in reusable iModel.

**Review by austeja-bentley — APPROVED** [review]

**Pull request merged**